vignettes/articles/run-stress-test.Rmd

```diff
@@ -0,0 +1,170 @@
+---
+title: "Run standard transition risk stress test"
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Running a standard climate transition risk stress test
+
+This vignette describes in detail, how to run a climate transition risk stress
+test for loans, using the function `run_stress_test` to calculate the credit risk
+of a loan book.
+
+## Assumptions
+
+In order to keep this vignette concise, we assume the following has been set up:
+
+1. R and RStudio are installed
+2. All required R packages are installed, this specifically includes
+`r2dii.data`, `r2dii.match`, `r2dii.analysis` and `r2dii.climate.stress.test`
+1. The user has obtained all relevant input files as described in the pdf manual
+1. The input files are organized in a folder structure consistent with the manual
+1. This notably implies the user has successfully run at least the matching
+part of PACTA for banks to produce the relevant project-specific input files
+1. The user has set up objects that point to the input directories for
+project-specific and project-agnostic input files.
+1. The user has successfully run the function `run_prep_calculation_loans()` to
+generate the input files ´for the main stress test function in the required
+format. Refer to the vignette **LINK** for details on how to do this.
+
+## Running the Analysis
+
+The climate transition risk stress test can be run for a single set of parameters
+or as a sensitivity analysis for a set of parameters that iterates over one of
+the input arguments. We will look at both cases.
+
+Use `library()` to attach the package
+
+```{r setup}
+library(r2dii.climate.stress.test)
+```
+
+## Running the Stress Test for one scenario
+
+In order to run a stress test for loans on one set of parameters, the user needs
+to pass exactly one value per input argument to the function
+`run_prep_calculation_loans()`. The only argument that doesnot have a default
+value and is absolutely required to be set by the user is `asset_type`. In our
+case, we want to run the stress test for loans, so the easiest way to obtain
+results is by running:
+
+```{r}
```

Comment:
compare above

Reply:
I don't understnd this one

Reply:
i said earlier that eval=FALSE should be added instead of commenting out the function calls

Reply:
done via overall options